patches/0001-Docs-changes.patch

```diff
@@ -0,0 +1,166 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Ian Wahbe <ian@wahbe.com>
+Date: Tue, 24 Oct 2023 15:23:27 -0700
+Subject: [PATCH 1/3] Docs changes
+
```

Comment:
nit: I don't think need to have these headers?

Reply:
This is the prefix that `git format-patch` produces. I'm not aware of a way to remove it that doesn't break `git am`.

Reply:
hmmm... maybe we shouldn't be using `git am` then; oh well, can deal with that at another time.